surprise/__init__.py

```diff
@@ -47,4 +45,4 @@
     "model_selection",
 ]
 
-__version__ = get_distribution("scikit-surprise").version
```

Comment:
```suggestion
__version__ = get_distribution("scikit-surprise").version

Reply:
Add back get_distribution().  I don't think you want to hardcode the version number here.

Reply:
Here, there are 2 considerations:

1. If we rely on `pkg_resources` (which comes from `setuptools`) inside `surprise`, then it becomes a direct dependency of `surprise`. Depending on the python distribution, `setuptools` may or may not be bundled with python. Hence to avoid this, I declare version directly in `surprise.__init__.py`
2. Most popular packages, like `scikit-learn` follow the same style: https://github.com/scikit-learn/scikit-learn/blob/main/sklearn/__init__.py

Let me know if this makes sense 🙂

Reply:
OK, thanks for the explanation.  That makes sense. 👍 
In that case, I would recommend  changing the docstring in setup.py 
in the section `Change __version__ ` to reflect that the version is now in `__init__.py`.

This is a valid change, because in `pyproject.toml` you specified that `version` is `dynamic`.